sw/device/benchmarks/coremark/top_earlgrey/ee_printf.c

```diff
@@ -663,6 +663,10 @@ int ee_printf(const char *fmt, ...) {
   va_end(args);
   p = buf;
   while (*p) {
+    if (*p == '\n') {
+      uart_send_char('\r');
+    }
```

Comment:
Why is that needed?

Reply:
The plain \n going over the serial results in the next line being immediately below the end of the previous one. \r is required to return to the beginning of the line hence a '\n' -> '\r\n' transformation is needed.

I did initially do this in ee_vsprintf but felt it made more sense here (as it's only an artifact is sending the string to the uart so the transformation should be done there rather than on the string generated by ee_vsprintf).

Reply:
That behavior probably depends on your serial terminal viewer, but good enough.

Reply:
True, screen definitely wants it. I think it doesn't matter to include it when it's not needed